source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md

```diff
@@ -0,0 +1,215 @@
+
+- `.ci/test/visual-regression/backstopConfig.js` is the [BackstopJS](https://github.com/garris/BackstopJS) configuration file. 
+
+1. Update the settings in `.ci/test/visual-regression/backstopConfig.js` file for your project. 
+
+    - For example, the `pathsToTest` variable determines the URLs to test.
+
+## Behat Testing 
+
+Behat testing uses `.ci/test/behat` and `tests/behat`. [Behat](https://behat.org/en/latest/) is an acceptance/end-to-end testing framework written in PHP. It facilitates testing the fully-built WordPress site on Pantheon. [WordHat](https://wordhat.info/) is used to help integrate Behat and WordPress.
+
+- `.ci/test/behat/initialize` deletes any existing WordPress user from Behat testing and creates a backup of the environment to be tested.
+
+- `.ci/test/behat/run` sets the `BEHAT_PARAMS` environment variable with dynamic information necessary for Behat and configures it to use [WP-CLI](https://wp-cli.org/) via [Terminus](/terminus). This script also creates the necessary WordPress user, starts headless Chrome, and runs Behat.
+
+- `.ci/test/behat/cleanup` restores the previously made database backup, deletes the WordPress user created for Behat testing, and saves screenshots taken by Behat.
+
+- `tests/behat/behat-pantheon.yml` runs tests against the Pantheon site.
+
+- `tests/behat/tests/behat/features` stores Behat `.feature` extension test files. 
+
+1. Store all `.feature` extension test files in the `tests/behat/tests/behat/features` directory.
+
+    - The example tests must be replaced with project-specific tests.
+
+        - `tests/behat/tests/behat/features/visit-homepage.feature` is a Behat test file that visits the homepage and verifies a `200` response.
+
+        - `tests/behat/tests/behat/features/admin-login.feature` is a Behat test file that logs into the WordPress dashboard as an administrator and verifies access to new user creation.
+
+        - `tests/behat/tests/behat/features/admin-login.feature` is a Behat test file that logs into the WordPress dashboard as an administrator, updates the `blogname` and `blogdescription` settings, clears the Pantheon cache, visits the home page, and verifies how the updated blog name and description appear.
+
+
+## GitHub Actions
+
+This section provides information enabling GitHub Actions for your site. 
+
+The Build Tools Site will configure GitHub Actions automatically if it was passed as the selected CI when creating the site. You will need to consult advanced external resources if you're working with an existing non-Build Tools site and want to add Github Actions.
+
+The steps to enable GitHub Actions for an existing Build Tools site created with another CI (e.g. CircleCI) may work for you.  
+
+1. Copy `.ci/.github` to `.github`. 
+
+1. Add the following secrets to the Github Actions configuration:
+
+    `ADMIN_EMAIL`
+    `ADMIN_PASSWORD`
+    `ADMIN_USERNAME`
+    `TERMINUS_TOKEN`
+    `TERMINUS_SITE`
+    `SSH_PRIVATE_KEY`
+    `GH_TOKEN`
```

Comment:
We should see if we can get build:project:repair to switch CI providers. Until that's confirmed, the advice above looks fine.